utility/doc/trustless_relay_validation.md

```diff
@@ -0,0 +1,65 @@
+# Validation of Trustless Relays
```

Comment:
Can you add a relative link to it in `utility/doc/README.md` so its discoverable?

Reply:
Thanks, fixed.